start-deployment-manager.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+set -e -u -x -o pipefail
+
+cd "$(dirname "$0")"
+
+if [ "$#" -ne 2 ]; then
+    echo "Wrong number of arguments. Usage: <Assembly postfix> <deployment manager project directory>"
+fi
+
+ASSEMBLY_POSTFIX="$1"
+LAUNCH_CONFIGURATION="$2"
+
+echo ${ASSEMBLY_POSTFIX}
+
+cp ${LAUNCH_CONFIGURATION} workers/deployment_manager/DeploymentManager/default_launch.json
+
+pushd "workers/deployment_manager"
+  sh publish-linux-workers.sh
+popd
+
+spatial build build-config
+spatial cloud upload "deployment_manager_${ASSEMBLY_POSTFIX}" --force
+spatial cloud launch "deployment_manager_${ASSEMBLY_POSTFIX}" deployment_manager_launch.json "deployment_manager_${ASSEMBLY_POSTFIX}" --snapshot=snapshots/empty.snapshot
```

Comment:
How come we copied a launch config over to `workers/deployment_manager/DeploymentManager/default_launch.json` and then here we're using `deployment_manager_launch.json` ?

Reply:
cause this is the launch configuration that starts the deployment of the deployment manager, not the deployments that get created by deployment manager